contact.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>PILAR TORRES</title>
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="index.html">
+  <link rel="stylesheet" href="portfolio.html">
+</head>
+
+<body>
+
+  <header>
+    <h1><a href="index.html">Pilar TORRES</a></h1>
+    
+    <nav class="navegation-bar">
+      <ul class="navegation-list">
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+        <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+
+    <h3>Contact me</h3>
+
+    <div class="container">
+      <form>
+        <label for="fname">First Name</label>
+        <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+        <label for="lname">Last Name</label>
+        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+        <label for="country">Country</label>
+        <select id="country" name="country">
+            <option value="australia">Spain</option>
```

Comment:
In select options, you need to make sure that the value matches somehow the text that is shown to the user. In this case, the value `australia` is not the text of this option, which is "Spain"